db/migrate/20240717140309_rename_chat_preferred_mobile_index.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class RenameChatPreferredMobileIndex < ActiveRecord::Migration[7.1]
```

Comment:
You should use the site setting rename generator - https://dev.discourse.org/t/-/57006#migrations-rename-site-setting-44